docs/conf.py

```diff
@@ -55,6 +55,9 @@
               'sphinx.ext.intersphinx',
               'sphinx.ext.doctest' ]
 
+autodoc_default_flags = ['members']
```

Comment:
Add also inherited members:
```suggestion
autodoc_default_flags = ['members', 'inherited-members']
```